### PR TITLE
Fix heading structure in sourcemaps troubleshooting

### PR DESCRIPTION
**Pull request by lforst**
Title

### CODE REVIEW

File: src/platform-includes/sourcemaps/troubleshooting/javascript.mdx [hunk 1]
```diff
@@ -57,19 +57,19 @@ If it doesn't, let us know by filing an issue on our [Sentry CLI repo](https://g
...
 
-## Verify You Are Running A Production Build
+### Verify You Are Running A Production Build
 
 When running JavaScript build tools (like webpack, Vite, ...) in development-mode/watch-mode, the generated code is sometimes incompatible with our source map uploading processes.
 
 We recommend, especially when testing locally, to run a production build to verify your source maps uploading setup.
 
-## Verify Your Source Files Contain Debug ID Injection Snippets
+### Verify Your Source Files Contain Debug ID Injection Snippets
```
Comment (lizokm) [inline src/platform-includes/sourcemaps/troubleshooting/javascript.mdx:72]:
```suggestion
### Verify That Source Files Contain Debug ID Injection Snippets
```

File: src/platform-includes/sourcemaps/troubleshooting/javascript.mdx [hunk 2]
```diff
@@ -137,25 +137,25 @@ new sourceMap.SourceMapConsumer(rawSourceMap).then(function (smc) {
 
 If you have the same (incorrect) results locally as you do via Sentry, double-check your source map generation configuration.
 
-## Verify your source files are not too large
+### Verify your source files are not too large
```
Comment (lizokm) [inline src/platform-includes/sourcemaps/troubleshooting/javascript.mdx:140]:
```suggestion
### Verify That Source Files Aren't Too Large
```

File: src/platform-includes/sourcemaps/troubleshooting/javascript.mdx [hunk 3]
```diff
@@ -137,25 +137,25 @@ new sourceMap.SourceMapConsumer(rawSourceMap).then(function (smc) {
...
 
 For an individual artifact, Sentry accepts a max filesize of **40 MB**.
 
 Often users hit this limit because they are transmitting source files at an interim build stage. For example, after your bundler has combined all your source files, but before minification has taken place. If possible, send the original source files.
 
-## Verify artifacts are not gzipped
+### Verify artifacts are not gzipped
```
Comment (lizokm) [inline src/platform-includes/sourcemaps/troubleshooting/javascript.mdx:146]:
```suggestion
### Verify That Artifacts Aren't Gzipped
```

File: src/platform-includes/sourcemaps/troubleshooting/javascript.mdx [hunk 4]
```diff
@@ -137,25 +137,25 @@ new sourceMap.SourceMapConsumer(rawSourceMap).then(function (smc) {
...
 
 The Sentry API currently only works with source maps and source files that are uploaded as plain text (UTF-8 encoded). If the files are uploaded in a compressed format (for example, gzip), they will be not be interpreted correctly.
 
 If you are using `sentry-cli` to upload your artifacts, starting with version `2.4.0` you can add the `--decompress` flag to your `sourcemaps upload` or `files upload` commands.
 
 Sometimes build scripts and plugins produce pre-compressed minified files (for example, webpack's [compression plugin](https://github.com/webpack/compression-webpack-plugin)). In these cases, you'll need to disable such plugins and perform the compression **after** the generated source maps/source files have been uploaded to Sentry.
 
-## Verify workers are sharing the same volume as web (if running self-hosted Sentry via Docker)
+### Verify workers are sharing the same volume as web (if running self-hosted Sentry via Docker)
```
Comment (lizokm) [inline src/platform-includes/sourcemaps/troubleshooting/javascript.mdx:154]:
```suggestion
### Verify Workers Are Sharing Same Volume as Web (if running self-hosted Sentry via Docker)
```

File: src/platform-includes/sourcemaps/troubleshooting/javascript.mdx [hunk 5]
```diff
@@ -137,25 +137,25 @@ new sourceMap.SourceMapConsumer(rawSourceMap).then(function (smc) {
...
 
 Sentry does source map calculation in its workers. This means the workers need access to the files uploaded through the front end. Double check that the cron workers and web workers can read/write files from the same disk.
 
-## Verify you are not using the `source-map-support` package
+### Verify you are not using the `source-map-support` package
```
Comment (lizokm) [inline src/platform-includes/sourcemaps/troubleshooting/javascript.mdx:158]:
```suggestion
### Verify You Aren't Using the `source-map-support` Package
```

**Review by lizokm — APPROVED** [review]
LGTM

**Comment by lforst** [comment]
@lizokm Thanks for taking a look! Idk why but the rest of the suggestions feel odd to me. We are not talking about any source files. We are talking about the user's source files. It feels like it is important here to convey that message.

**Pull request merged**